markets/perps-market/test/integration/Orders/OffchainAsyncOrder.commit.test.ts

```diff
@@ -246,21 +212,19 @@ describe('Commit Offchain Async Order test', () => {
       });
 
       describe('can settle order', () => {
-        settleOrder(
-          {
-            keeper: keeper,
-            marketId: () => ethMarketId,
-            accountId: () => 2,
-            feedId: () => feedId,
-            startTime: () => startTime,
-            settlementDelay: () => settlementDelay,
-            offChainPrice: () => 1000,
-          },
-          {
+        before('settle', async () => {
+          const settlementTime = startTime + DEFAULT_SETTLEMENT_STRATEGY.settlementDelay + 1;
+          await fastForwardTo(settlementTime, provider());
```

Comment:
this fast forward orchestration happens in the openPosition but when we're testing the actual commit and settle, we should do the time fast forward orchestration in the test so its clear imo. thoughts?

Reply:
This test is focusing on the commit, that was originally used the settleHelper to help confirm the order can be settled. Settlement is tested fully (wip) in another test